docs/tutorials/actions.mdx

```diff
@@ -114,8 +117,8 @@ cardano-cli conway governance action update-committee \
   --testnet \
   --governance-action-deposit 1000000000 \
   --deposit-return-stake-verification-key-file stake.vkey \
-  --anchor-url https://tinyurl.com/3wrwb2as \
-  --anchor-data-hash 52e69500a92d80f2126c836a4903dc582006709f004cf7a28ed648f732dff8d2 \
+  --anchor-url https://github.com/Ryun1/metadata/blob/main/cip108/treasury-withdrawal.jsonld \
+  --anchor-data-hash 931f1d8cdfdc82050bd2baadfe384df8bf99b00e36cb12bfb8795beab3ac7fe5 \
   --remove-cc-cold-verification-key-hash 89181f26b47c3d3b6b127df163b15b74b45bba7c3b7a1d185c05c2de \
   --quorum 1/2 \
   --governance-action-tx-id fe2c99fe6bc75a9666427163d51ae7dbf5a60df40135361b7bfd53ac6c7912ec \
```

Comment:
These 2 have to be changed to  `--prev-governance-action-tx-id` and `--prev-governance-action-index`

Reply:
done!
thanks for the spot @Hornan7